k8s.io/ingress-canary-v6.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Don't we want to move to "v1" rather than "v1beta1" ?

Reply:
@thockin sorry for the time taken to answer. I missed your comment. Looking at :
```ameukam@cloudshell:~ (kubernetes-public)$ kubectl version 
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.4", GitCommit:"e87da0bd6e03ec3fea7933c4b5263d151aafd07c", GitTreeState:"clean", BuildDate:"2021-02-18T16:12:00Z", GoVersion:"go1.15.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.15-gke.7800", GitCommit:"cef3156c566a1d1a4b23ee360a760f45bfbaaac1", GitTreeState:"clean", BuildDate:"2020-12-14T09:12:37Z", GoVersion:"go1.13.15b4", Compiler:"gc", Platform:"linux/amd64"}
ameukam@cloudshell:~ (kubernetes-public)$ kubectl api-resources | grep 'ingress'
ingresses                          ing             extensions/v1beta1                     true         Ingress
ingresses                          ing             networking.k8s.io/v1beta1              true         Ingress
```

we can't switch to "v1". Will do it once `aaa` is upgraded to v1.19.